src/components/badge/index.tsx

```diff
@@ -104,6 +104,10 @@ export type BadgeProps = ViewProps &
      * Use to identify the badge in tests
      */
     testId?: string;
+    /**
+     * Component accessibility label
+     */
+     accessibilityLabel?: string;
```

Comment:
No reason to declare this type. 
Badge is already has this cause its typings based on ViewProps